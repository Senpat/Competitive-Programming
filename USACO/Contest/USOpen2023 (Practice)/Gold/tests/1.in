1
8
7 5
2 1
4 2
5 1
3 2
8 5
6 2
4
8 5
5 1
6 5
