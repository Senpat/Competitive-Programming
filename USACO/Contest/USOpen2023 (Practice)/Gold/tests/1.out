4
2 5
4 8
3 8
7 8
